File: walk-time.vb
Module VBModule
    Sub Main()
        Dim hrs As Integer = CInt(Math.Ceiling(Rnd() * 3))
        Dim mins As Integer = CInt(Math.Ceiling(Rnd() * 60) - 1)
        Dim output As String 
        
        If mins >= 10 Then
            output = hrs.ToString() + ":" + mins.ToString() + "PM"
        Else
            output = hrs.ToString() + ":0" + mins.ToString() + "PM"
        End If
        
        Console.WriteLine(output)
    End Sub
End Module

File: walk-time.lua
math.randomseed(os.time())
hrs = math.random(3); 
mins = math.random(60) - 1;

if mins >= 10
then print(string.format("%d:%dPM",hrs,mins));
else print(string.format("%d:0%dPM",hrs,mins));
end
